src/main/webapp/data-visualization.js

```diff
@@ -508,6 +529,142 @@ function toggleDataTable() {
   }
 }
 
+function displayYearlyChart(chartId, data, description, location) {
+  const chart = am4core.create(chartId, am4charts.XYChart);
+  chart.data = data;
+  const dateAxis = chart.xAxes.push(new am4charts.DateAxis());
+  dateAxis.dataFields.category = 'year';
+  dateAxis.title.text = 'Year';
+  dateAxis.baseInterval = {timeUnit: 'year'};
```

Comment:
I'm not sure where to bring this up, but I'd like if the charts had a bit more height. They're very wide and short, and I think it'd be interesting to stretch out the y-axis a little more (maybe that could be done around these lines of code?) and see how that looks.

Reply:
Changed height from 300px to 500px in CSS

Reply:
LGTM